generator/src/test/java/com/scottlogic/deg/generator/decisiontree/TreeTransformationIntegrationTests.java

```diff
@@ -117,7 +118,7 @@
     }
 
     private Profile getProfile(Path path) throws IOException, InvalidProfileException {
-        return new ProfileReader().read(path);
+        return new ProfileReader().read(path, GenerationConfig.Constants.DEFAULT_PROFILE_VALIDATION);
```

Comment:
Should validation be explicitly disabled here?

Reply:
I don't think the tests have anything to do with the profile validation so I think it should be disabled. Re: explicity, perhaps it will be better to create a method overload in the profile reader where it is set to disabled in just one place. Is that what you meant? Thanks.

Reply:
I think its fine with the method signature as it is; but be explicit about whether validation is on or not.

Reply:
changed.